src/CheckoutSdk/Workflows/Four/Events/EventTypesResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Checkout.Workflows.Four.Events
+{
+    public class EventTypesResponse
```

Comment:
What do you think on extends `Event` given the properties are the same on this class?